rapid7_vulndb/komand_rapid7_vulndb/connection/connection.py

```diff
@@ -11,11 +11,8 @@ def __init__(self):
         super(self.__class__, self).__init__(input=ConnectionSchema())
 
     def connect(self, params):
-        pass
-
-    def test(self):
-        result = requests.get("https://www.rapid7.com/db/search")
-        if (result.status_code == requests.codes.ok) is True:
+        result = requests.get("https://vdb-kasf1i23nr1kl2j4.rapid7.com/v1/info")
+        if result.status_code == requests.codes.ok:
```

Comment:
Another option this is the `raise_for_status` method: https://requests.readthedocs.io/en/master/api/#requests.Response.raise_for_status

Reply:
Disregard, saw that you use it later :)